data/sql/updates/pending_db_world/rev_1667732623534207400.sql

```diff
@@ -0,0 +1,20 @@
+-- Fire brigade esES / esMX
+INSERT INTO quest_offer_reward_locale (ID, locale, RewardText, VerifiedBuild) VALUES
```

Comment:
Please DELETE first before you insert records so that the script can be reused.


Also please add backticks around all column names for a better codestyle.
```suggestion
INSERT INTO `quest_offer_reward_locale` (`ID`, `locale`, `RewardText`, `VerifiedBuild`) VALUES
```